lighthouse-labs/discord-shift-swap
# Trade v1

## Change Summary
This is the initial creation of the discord shift swap bot for use by mentors.

## Setup Instructions
Instructions can be found in the readme.md file.

## Issue
N/A
Resolves `none`

## Estimation
Estimated: 1w
Actual: 8d

## Manual Testing
Join a testing Discord server and make sure the bot is in there. Run the code and run all commands, starting with `/trade`.

## Documentation
1. Got the bot to work initially for testing.
2. Changed the start command to only allow users certain options that are needed.
3. Changed `start` command to `trade`.
4. Added LHL branding and removed all of IVON's branding.
5. Code cleanup/removed a lot of unnecessary commands/branding/code.
